### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by iamemilio**
# Changes
Setting up the data connection object in the service JSON object in the following format will allow you to send tokens to pfioh servers.
```
 'data': {
    'addr':            '%PFIOH_IP:5055',
    'baseURLpath':     'api/v1/cmd/',
    'status':          'undefined',
    "enableTokenAuth": True,
    "authTokens":      "password"
   },
```
All authToken fields should be in plain text
This can also send Tokens to pman, but it is not fully fleshed out yet. 
@danmcp @rudolphpienaar

### CODE REVIEW

File: pfcon/pfcon.py [hunk 1]
```diff
@@ -116,16 +118,30 @@
...
                 "status":                       "undefined"
             },
             "data": {
                 "addr":                         "pfioh-myproject.127.0.0.1.nip.io",
                 "baseURLpath":                  "api/v1/cmd/",
-                "allowUnverifiedCertificates":  False,  
                 "status":                       "undefined"
             }
+        },
+        "openshiftlocal-tokens": {
```
Comment (danmcp) [inline pfcon/pfcon.py]: @betaredux Why is this here?  Is this just here for testing?

Comment (betaredex) [reply pfcon/pfcon.py]: It doesn't do anything as far as I can tell. It would probably be better to ask @iamemilio 
(Also, you misspelled my username).

Comment (iamemilio) [reply pfcon/pfcon.py]: Hi, this was just for testing pfcon against pfioh with auth enabled in openshift. It is fine to remove if you would like.
